besu/src/main/java/org/hyperledger/besu/services/RpcEndpointServiceImpl.java

```diff
@@ -48,6 +65,37 @@ public <T> void registerRPCEndpoint(
     rpcMethods.put(namespace + "_" + functionName, function);
   }
 
+  @Override
+  public PluginRpcResponse call(final String methodName, final Object[] params) {
```

Comment:
Should we LOG.trace the methodName and params here?

Reply:
done